tests/compile-fail/ptr_offset_int_plus_ptr.rs

```diff
@@ -0,0 +1,9 @@
+// error-pattern: pointer value as raw bytes
+
+fn main() {
+    let ptr = &mut *Box::new(0u32) as *mut u32;
```

Comment:
`Box::into_raw`?

Reply:
I guess that's indeed shorter, yes. I don't like it very much because it is not a method.